### DISCUSSION TITLE
AlDente for Magic Keyboard/Trackpad

### DISCUSSION THREAD

**Discussion by jqrs** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1010] [Ideas]
Although this is out of scope for the focus of AlDente, it would be useful to limit the charge of attached Magic Keyboard and Trackpad devices in the same way AlDente does for Macbooks.

Feel free to remove if this is not the place; not sure where else to post this.

**Comment by MatthiasKerbl** [comment]
Hi @mojoba,

Love the idea, however, it might be difficult or not even possible to alter the charging behavior of the Magic Keyboard and Trackpad.
Since this is not directly related to AlDente, I will move it to Discussions.

Best,
Matthias
